sql/core/src/test/resources/sql-tests/inputs/postgreSQL/interval.sql

```diff
@@ -0,0 +1,330 @@
+--
+-- Portions Copyright (c) 1996-2019, PostgreSQL Global Development Group
+--
+--
+-- INTERVAL
+-- https://github.com/postgres/postgres/blob/REL_12_STABLE/src/test/regress/sql/interval.sql
+
+-- SET DATESTYLE = 'ISO';
```

Comment:
Although we don't support this technically, shall we uncomment this since `SET` comment accepts this?

Reply:
If Spark doesn't have such settings, does it make sense to set this? I am not sure about this.

Reply:
This should be covered by https://issues.apache.org/jira/browse/SPARK-28259 . I will add a reference to the issue.